build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+mix deps.get
+
+mix ecto.create
+mix ecto.migrate
+
+mix test
```

Comment:
@teerawat1992 shall we run test when building project?

Reply:
No need